src/qt/rpcconsole.h

```diff
@@ -118,6 +118,10 @@ public Q_SLOTS:
     void updateMasternodeCount();
     /** Set number of blocks, last block date and last block hash shown in the UI */
     void setNumBlocks(int count, const QDateTime& blockDate, const QString& blockHash, double nVerificationProgress, bool headers);
+    /** Set hash of latest ChainLocked block */
+    void setBestChainLockHash(std::string blockHash);
```

Comment:
Instead of using two methods, should probably make this look more like 
`void setNumBlocks(int count, const QDateTime& blockDate, const QString& blockHash, double nVerificationProgress, bool headers);`